README.md

```diff
@@ -24,12 +26,12 @@ npx gatsby new gatsby-ready-to-go https://github.com/Pagepro/gatsby-ready-to-go
 
 If you prefer you can clone this repository and run the following commands inside the project folder:
 
-1. `npm install` or `yarn`;
-2. `yarn start`;
+1. `npm install` or `yarn`
+2. `yarn start`
 
 To view the project you can open `http://localhost:8000`.
 
-## Other Available Scripts
+## Other available scripts
 
 `yarn storybook`
 
```

Comment:
Maybe add some instructions how to use plop?

Reply:
And also some info about imports order

Reply:
@jdakowicz it's already exist:
![image](https://user-images.githubusercontent.com/18337813/118341324-e3c64400-b51e-11eb-990a-8d004de9294e.png)

@joanna-pagepro added.